site/constants/index.ts

```diff
@@ -30,3 +30,11 @@ export const DISRUPTION_REASONS: {
         reason: "Special Event",
     },
 ];
+
+export const COOKIES_POLICY_COOKIE = "fdbt-cookies-policy";
```

Comment:
fdbt stands for fares data build tool - we probably can just call this disr-cookies-policy or similar

Reply:
changed